Makefile

```diff
@@ -1,115 +1,96 @@
 .DEFAULT_GOAL := help
+.PHONY: help clean build dist
```

Comment:
the reason I went with multiple PHONY is that it's easy to check you didn't forget one. You see a rule and you can see if it's a command by the presence of the PHONY target. Without it you easily add a new command and completely forget about it

Reply:
Yeah, I can see that. But you could also add a new command down at the bottom of the file, and never thing about adding .PHONY because none of the targets down there have it :) 

I feel like having cleaner command definitions outweighs downsides.

I did just move the .PHONY and .DEFAULT_GOAL next to the list of commands, since that's what they're used for. That is probably a good balance?

Reply:
I think it's ok either way. You don't have that many commands so it's ok, I've been bitten a couple of times by some missing `.PHONY` target and easier to manage the other way but I also had more commands, so harder to track